crates/tauri-runtime-wry/src/monitor/mod.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2024 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+use tao::dpi::PhysicalSize;
+
+#[cfg(any(
+  target_os = "linux",
+  target_os = "dragonfly",
+  target_os = "freebsd",
+  target_os = "netbsd",
+  target_os = "openbsd"
+))]
+mod linux;
+#[cfg(target_os = "macos")]
+mod macos;
+#[cfg(windows)]
+mod windows;
+
+#[cfg(desktop)]
+pub trait MonitorExt {
+  /// Enable or disable the window
+  ///
+  /// ## Platform-specific:
+  ///
+  /// - **Android / iOS**: Unsupported.
+  fn get_work_area_size(&self) -> PhysicalSize<u32>;
```

Comment:
let's rename to just 
```suggestion
  fn work_area(&self) -> PhysicalSize<u32>;
```